quarkus/template.yaml

```diff
@@ -14,6 +14,7 @@ Globals:
     MemorySize: 1024
     Environment:
       Variables:
+        JAVA_TOOL_OPTIONS: "-XX:+TieredCompilation -XX:TieredStopAtLevel=1"
```

Comment:
This is a welcome update. But, I don't think Micronaut is using tired compilation either. Can you do the same for Micronaut?